.github/workflows/health.yaml

```diff
@@ -0,0 +1,71 @@
+# A CI configuration to check PR health.
+
+name: Health
+
+# Callers of this workflow should use it as follows:
+#
+# name: Health
```

Comment:
I think it'd be worth repeating some of this info ('how to use') in a brief section in the readme (ala what we have now for the publishing automation).